sdk/communication/communication-administration/src/communicationIdentity/communicationIdentityClient.ts

```diff
@@ -28,6 +29,7 @@ import {
 } from "./models";
 import { VoidResponse } from "../common/models";
 import { attachHttpResponse } from "../common/mappers";
+import { bearerTokenAuthenticationPolicy, BearerTokenAuthenticationPolicy } from "@azure/core-http/types/latest/src/policies/bearerTokenAuthenticationPolicy";
```

Comment:
`BearerTokenAuthenticationPolicy` is never used, CI actually fails because of it.

We never import from a deep module link, import `bearerTokenAuthenticationPolicy` from top-level "@azure/core-http" together with the other modules in line 11-16

Reply:
Removed the import